pb/api/discovery.proto

```diff
@@ -33,11 +33,11 @@ option (gogoproto.gostring_all) = true;
 option (gogoproto.marshaler_all) = true;
 option (gogoproto.unmarshaler_all) = true;
 
-// DiscoveryService is a service which is used to discover service, service config
-// and service endpoints.
+// DiscoveryService is a service which is used to discover service, service
+// config and service endpoints.
 service DiscoveryService {
-  rpc StreamSvcs(SvcDiscoveryRequest)
-      returns (stream SvcDiscoveryResponse) {}
+  rpc StreamDependency(DependencyRequest)
```

Comment:
1. better to use plural `StreamDependencies`
2. `DependencyRequest` -> `DependencyDiscoveryRequest`